ui/client/components/graph/NodeDetailsContent.js

```diff
@@ -449,14 +450,26 @@ export class NodeDetailsContent extends React.Component {
       <div className="node-row">
         {this.renderFieldLabel(fieldLabel)}
         <div className={nodeValueClass}>
-          <ExpressionSuggest fieldName={fieldName} inputProps={{
-            rows: 1, cols: 50, className: "node-input", value: expressionObj.expression, language: expressionObj.language,
-            onValueChange: ((newValue) => this.setNodeDataAt(exprTextPath, newValue)), readOnly: false}}/>
+          {this.createExpressionComponent((newValue) => this.setNodeDataAt(exprTextPath, newValue), fieldName, expressionObj)}
         </div>
       </div>)
     )
   };
 
+  createExpressionComponent = (onValueChange, fieldName, expressionObj) => {
```

Comment:
Why you assume that plugin will load only at createExpressionComponent? Mayby we should think about more generic method..?

Reply:
changed a bit